src/exception.jl

```diff
@@ -12,15 +26,6 @@ struct PyError <: Exception
     # generate a PyError object.  Should normally only be called when
     # PyErr_Occurred returns non-NULL, and clears the Python error
     # indicator.
-    function PyError(msg::AbstractString)
-        ptype, pvalue, ptraceback = Ref{PyPtr}(), Ref{PyPtr}(), Ref{PyPtr}()
-        # equivalent of passing C pointers &exc[1], &exc[2], &exc[3]:
-        ccall((@pysym :PyErr_Fetch), Cvoid, (Ref{PyPtr},Ref{PyPtr},Ref{PyPtr}), ptype, pvalue, ptraceback)
-        ccall((@pysym :PyErr_NormalizeException), Cvoid, (Ref{PyPtr},Ref{PyPtr},Ref{PyPtr}), ptype, pvalue, ptraceback)
-        new(msg, PyObject(ptype[]), PyObject(pvalue[]), PyObject(ptraceback[]))
-    end
-
-    PyError(msg::AbstractString, e::PyError) = new(msg, e.T, e.val, e.traceback)
```

Comment:
Why delete these constructors?

Reply:
I check the value returned by `PyErr_Fetch` and depending on the output I build either a `PyError` or a `PyJLError`. Because of this, I figured that moving those calls outside of the constructor would be a better idea.